addons/website_event/controllers/main.py

```diff
@@ -185,6 +185,7 @@ def _prepare_event_register_values(self, event, **post):
             'range': range,
             'google_url': lazy(lambda: urls.get('google_url')),
             'iCal_url': lazy(lambda: urls.get('iCal_url')),
+            'failed_state': post.get('failed_state'),
```

Comment:
Ok I think I see it now, can we rename to "registration_error_code" or something like that then?